plugin/trino-postgresql/src/main/java/io/trino/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -224,9 +224,7 @@
     private static final int ARRAY_RESULT_SET_VALUE_COLUMN = 2;
     private static final String DUPLICATE_TABLE_SQLSTATE = "42P07";
     private static final int POSTGRESQL_MAX_SUPPORTED_TIMESTAMP_PRECISION = 6;
-
-    // TODO: Since 42.2.23 the JDBC driver will return 0 as column size for decimal types with unspecified precision
-    private static final int PRECISION_OF_UNSPECIFIED_DECIMAL = 131089;
+    private static final int PRECISION_OF_UNSPECIFIED_DECIMAL = 0;
```

Comment:
I'd be so happy with myself if this is the only change needed to get DECIMALs to work correctly.